webapp/angular.json

```diff
@@ -100,7 +100,8 @@
               "src/tsconfig.spec.json"
             ],
             "exclude": [
-              "**/node_modules/**"
+              "**/node_modules/**",
+	      "**/*.json"
```

Comment:
Why did you need that? `json` files won't get build anyway if I'm correct.

Reply:
that's in the `lint` section. And the linter tries to lint the json-files...